corehq/apps/export/views.py

```diff
@@ -2489,7 +2489,11 @@ def can_download_daily_saved_export(export, domain, couch_user):
 @require_GET
 def download_daily_saved_export(req, domain, export_instance_id):
     with CriticalSection(['export-last-accessed-{}'.format(export_instance_id)]):
-        export_instance = get_properly_wrapped_export_instance(export_instance_id)
+        try:
+            export_instance = get_properly_wrapped_export_instance(export_instance_id)
+        except ResourceNotFound:
+            raise Http404
```

Comment:
nit: Can we update this to `Http404(_(u"Export not found"))` so its clear that the url is fine but we are missing the resource?

Reply:
sure - 0d6bc95

Reply:
Thank you.